docs/release_notes/0.119.0.md

```diff
@@ -0,0 +1,29 @@
+# Release v0.119.0
+
+## 🚀 Features
+
+- Add resolve conflicts option on addon create (#5880)
+
+## 🎯 Improvements
+
+- Improve error message when Fargate is unsupported in an AZ (#5875)
+
+## 🐛 Bug Fixes
+
+- Fix publish-docs workflow (#5901)
```

Comment:
this should be listed under maintenance imo

Reply:
https://github.com/weaveworks/eksctl/pull/5904/commits/2b9bb408a3b2d7012b1e760177a5dffb53f8fb9c